includes/options/class-amp-options-menu.php

```diff
@@ -670,6 +681,88 @@ public function render_stories_export() {
 		<?php
 	}
 
+	/**
+	 * Render the stories settings section.
+	 *
+	 * @since 1.3
+	 */
+	public function render_stories_settings() {
+		$this->render_stories_settings_page_advance();
+	}
+
+	/**
+	 * Render the page advancement settings section.
+	 *
+	 * @since 1.3
+	 */
+	private function render_stories_settings_page_advance() {
+		$stories_settings_definitions = AMP_Story_Post_Type::get_stories_settings_meta_definitions();
+
+		/**
+		 * Currently the list of options with values and descriptions is hardcoded in PHP and JS.
+		 * There should be a single source for this data to be shared in the application.
+		 *
+		 * @todo Figure out the best way to keep these settings DRY.
```

Comment:
Since #3171 we have this new `window.ampStoriesEditorSettings` global and a `getSettings()` selector. Could we use that to pass these defaults to the editor?